motoserver/Tiltfile

```diff
@@ -21,10 +21,16 @@ def motoserver_up(init_script = "", **k8s_kwargs):
     else:
         init_script = read_file(os.path.join(extension_root, "scripts/init.sh"))
 
-    k8s_yaml([
-        os.path.join(extension_root, "k8s/moto-statefulset.yaml"),
-        os.path.join(extension_root, "k8s/moto-service.yaml"),
-    ])
+    statefulset = read_yaml(os.path.join(extension_root, "k8s/moto-statefulset.yaml"))
+    if not keep_state:
+        statefulset_env = statefulset["spec"]["template"]["spec"]["containers"][0]["env"]
+        for env_var in statefulset_env:
+            if env_var["name"] == "MOTO_ENABLE_RECORDING":
+                env_var["value"] = "False"
```

Comment:
Is this working? The code in main.py lines 85 and 91 are using lowercased "false".